build.gradle

```diff
@@ -138,6 +138,10 @@ tasks.withType(Checkstyle) {
     exclude 'uk/gov/hmcts/ccd/domain/types/sanitiser/document/**'
 }
 
+tasks.withType(Test) {
+    maxParallelForks = System.getenv('PARALLEL_THREADS') ?: 6
```

Comment:
Would we have PARALLEL_THREADS defined in Jenkins_CNP ?

Might be tempted to rename to MAX_NUM_PARALLEL_THREADS so that its clear that a number is required rather than a boolean yes/no ?

Reply:
Good point actually, will update this now